contractsV5/FeeHandler.sol

```diff
@@ -0,0 +1,200 @@
+pragma solidity 0.5.11;
+
+import "./IKyberDAO.sol";
+import "./IFeeHandler.sol";
+import "./PermissionGroupsV5.sol";
+import "./IKyberNetworkProxy.sol";
+import "./UtilsV5.sol";
+import "./IERC20.sol";
+import "./IBurnableToken.sol";
+
+contract FeeHandler is IFeeHandler, Utils {
+
+    IKyberDAO public kyberDAOContract;
+    // if I use the interface, I am not able to get the KyberNetwork internal contract from the proxy..
```

Comment:
I get your comment
the correct solution to this is create a local interface that can do this. and not include all the code.
Anyway now its set in ctor. so no problem.

Reply:
yep, will remove comment.